docs/reference/versions/index.md

```diff
@@ -0,0 +1,37 @@
+# Service version matrix
+
+Services in {{extra.brand}} Cloud are updated on a regular basis and
+on a rolling schedule.
+
+This section lists the cloud API service versions available in each
+{{extra.brand}} Cloud region.
+
+* [Public Cloud](public.md): versions running in our {{extra.brand}}
+  Public Cloud regions.
+
+* [Compliant Cloud](compliant.md): versions running in our
+  {{extra.brand}} Compliant Cloud regions.
+
+
+## OpenStack Services
+
+[OpenStack releases](https://releases.openstack.org/) are named, in
```

Comment:
This is very operational. I am not eager to show that in. Except if this is in SLA, which I don't think it is.

Reply:
see also my comment above. Marking this as resolved.